watertap/unit_models/mvc/tests/test_mvc.py

```diff
@@ -148,7 +148,7 @@ def test_mvc():
     solver = get_solver()
     initialize(m, solver=solver)
 
-    results = solver.solve(m, tee=False)
+    results = solver.solve(m, tee=True)
```

Comment:
Should we set back to False?

Reply:
No, because pytest suppresses the output by default, and if the test fails we always want to look at the solver logs.

Which is part of the reason I left some other debugging outputs in the tests as well. I don't see any point in *not* having them immediately when needed.